spec/models/pantograph_spec.rb

```diff
@@ -20,6 +20,13 @@
   # it { is_expected.to validate_uniqueness_of(:text) }
   # it { is_expected.to validate_uniqueness_of(:tweet_id) }
 
+  describe '#publish_next' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.